docs/embedding/sdk/appearance.md

```diff
@@ -13,9 +13,9 @@ You can style your embedded Metabase components with a theme.
 Here's an example that includes the various styling options available:
 
 ```js
-import {defineEmbeddingSdkTheme} from "@metabase/embedding-sdk-react";
+import {defineMetabaseTheme} from "@metabase/embedding-sdk-react";
```

Comment:
```suggestion
import { defineMetabaseTheme } from "@metabase/embedding-sdk-react";
```

Small adjustment. I wonder if there's a formatter for code blocks like this 🤔 This isn't something we should have to be aware of.